src/results/ashp.jl

```diff
@@ -0,0 +1,126 @@
+# REopt®, Copyright (c) Alliance for Sustainable Energy, LLC. See also https://github.com/NREL/REopt.jl/blob/master/LICENSE.
+
+"""
+`ASHP` results keys:
+- `size_mmbtu_per_hour`  # Thermal production capacity size of the ASHP [MMBtu/hr]
+- `electric_consumption_series_kw`  # Fuel consumption series [kW]
+- `annual_electric_consumption_kwh`  # Fuel consumed in a year [kWh]
+- `thermal_production_series_mmbtu_per_hour`  # Thermal energy production series [MMBtu/hr]
```

Comment:
Related but counter to changing the sizing capacity/basis to ton, I'm not opposed to reporting heating load production as mmbtu/hr because that's how we have other heating results. Let's just make sure we're using the same naming convention for thermal/heating/cooling results as other heating and cooling techs, like CHP and GHP. I think this is consistent, and we're differentiating between heating production and cooling production based on the units.

Reply:
Addressed here https://github.com/NREL/REopt.jl/commit/032e0d6cd659b47851b2a41c45bfe06c434afd4f